javascript.md

```diff
@@ -349,3 +349,22 @@ throw "Error message";    // throw error text to user
 |g|	Performs a global match and finds all|
 |i|	Performs case-insensitive matching|
 |m|	Performs multiline matching|
+
+## Closures
```

Comment:
Could you please wite this as a post instead?

Reply:
Can you pl elaborate

Reply:
This is a very specific use-case, and does not belong in Cheat Sheets, please write this as a post instead here https://onecompiler.com/posts